.github/workflows/brakeman-analysis.yml

```diff
@@ -0,0 +1,44 @@
+# This workflow integrates Brakeman with GitHub's Code Scanning feature
+# Brakeman is a static analysis security vulnerability scanner for Ruby on Rails applications
+
+name: Brakeman Scan
+
+# This section configures the trigger for the workflow. Feel free to customize depending on your convention
+on:
+  push:
+    branches: [ develop ]
+  pull_request:
+    branches: [ develop ]
+
+jobs:
+  brakeman-scan:
+    name: Brakeman Scan
+    runs-on: ubuntu-latest
+    steps:
+    # Checkout the repository to the GitHub Actions runner
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    # Customize the ruby version depending on your needs
+    - name: Setup Ruby
+      uses: actions/setup-ruby@v1
```

Comment:
I'm leaning towards using `ruby/setup-ruby` these days, as it takes the version of Ruby to use directly from the `.ruby-version` file

Reply:
Aha good call! Still learning how to do these actions 👍

Reply:
Also, if you add

```
with:
        bundler-cache: true
```

to the action, it automagically caches the bundle, speeding up subsuqent builds - https://github.com/ruby/setup-ruby#caching-bundle-install-automatically